extensions/amp-next-page/validator-amp-next-page.protoascii

```diff
@@ -52,5 +52,13 @@ tags: {  # <amp-next-page>
   tag_name: "AMP-NEXT-PAGE"
   unique: true
   requires_extension: "amp-next-page"
+  attrs: {
+    name: "src"
+    value_url: {
+      allowed_protocol: "https"
+      allow_relative: true  # Will be set to false at a future date.
```

Comment:
Let's set this to `false` now. Relative URLs have been causing random issues in viewer like 
#16683

Reply:
/to @honeybadgerdontcare

Reply:
Done.